CONTRIBUTING.md

```diff
@@ -0,0 +1,168 @@
+# How to contribute
+
+## Prerequisites
+
+- [Node.js][n] 16+;
+- [Git][g];
+- [Docker Desktop][d];
```

Comment:
I don't think docker should be a pre-requisite for working in this repo.

Reply:
I can make the Docker an optional dependency. It's feels right for you?

Reply:
Sure, I think if it's helpful for others, it's worth including as a note, but I don't think we should treat it as required.  Personally I'd suggest adding it in another PR, though, if the goal of this PR is documentation.

Reply:
Fixed.